src/api/apiFetch.js

```diff
@@ -52,15 +52,18 @@ export const apiCall = async (
     }
     const error = makeErrorFromApi(response.status, json);
 
+    const { httpStatus, data } = error;
+
     // eslint-disable-next-line no-console
-    console.log({ route, params, httpStatus: response.status, json });
+    console.log({ route, params, httpStatus, data });
     Sentry.addBreadcrumb({
       category: 'api',
       level: 'info',
-      data: { route, params, httpStatus: response.status, json },
+      data: { route, params, httpStatus, data },
```

Comment:
```
api errors [nfc]: Rely on error object in logging code.
```

nit: Breadcrumbs sent to Sentry will look different after this commit, so I think that makes it non-NFC.

Reply:
Mmm, indeed, good catch.

I think actually the name should ideally say something like "result" or "response" -- neither "json" nor "data" is real clear that this is the data the server sent back in the body, rather than say the data that we sent in the request. I'll pick a name like that.